src/System.Runtime.InteropServices.RuntimeInformation/tests/CheckPlatformTests.cs

```diff
@@ -1,26 +1,17 @@
-﻿using System.Runtime.InteropServices;
+using System.Runtime.InteropServices;
 using Xunit;
 
 namespace System.Runtime.InteropServices.RuntimeInformationTests
 {
     public class CheckPlatformTests
     {
-        [Fact, PlatformSpecific(PlatformID.Windows)]
-        public void CheckWindows()
-        {
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Windows));
-            Assert.True(RuntimeInformation.IsOSPlatform(OSPlatform.Create("WINDOWS")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("windows")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Create("Windows NT")));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.Linux));
-            Assert.False(RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
-        }
-
         [Fact, PlatformSpecific(PlatformID.Linux)]
```

Comment:
Where are the CheckFreeBSD()  tests?

        [Fact, PlatformSpecific(PlatformID.FreeBSD)]
        public void CheckFreeBSD()
        {
            // yada
        }